source/dagger/gameplay/racing/racing_game_logic.cpp

```diff
@@ -20,6 +21,23 @@ void RacingCollisionsLogicSystem::WindDown()
     Engine::Dispatcher().sink<NextFrame>().disconnect<&RacingCollisionsLogicSystem::OnEndOfFrame>(this);
 }
 
+void racing_game::CreateStrike(float TileSize,int life)// Vector3 pos_
+{
+    auto view = Engine::Registry().view<Transform, ControllerMapping, RacingPlayerCar>();
```

Comment:
do we need this view here?

Reply:
No we don't, sorry about that. I was trying something else and forgot to delete it.